test/verify/check-packagekit

```diff
@@ -392,7 +396,7 @@ class TestUpdates(MachineCase):
 
         # make sure we have enough time to crash PK
         self.createPackage("slow", "1", "1", install=True)
-        self.createPackage("slow", "1", "2",  postinst='sleep 10')
+        self.createPackage("slow", "1", "2",  postinst="while [ ! -e /tmp/finish-pk ]; do sleep 1; done")
```

Comment:
Indeed we don't actually need to finish the package update here, and the lock file never gets created (it doesn't need to). What about `sleep infinity` instead, to make this simpler?